recipes/virtualenv/build.sh

```diff
@@ -0,0 +1,4 @@
+
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
Figured there was something subtle with `virtualenv` hence the "if this is using `setuptools` to install...". 😄

So you shouldn't need the arguments. The other comment still applies.